tests/testthat/test_ForwardSelection.R

```diff
@@ -0,0 +1,29 @@
+test_ForwardSelection<- function(){
+
+  # Setting
+  y <- mtcars$mpg
+  x <- mtcars[,-1]
+  selected <- ForwardSelection(my_mod="lm", feature=x, label=y, type="regression", cv=5)
+
+  train_control <- caret::trainControl(method="cv", number=5)
+  ffs_model <- caret::train(x=x[selected], y=y, trControl = train_control, method ="lm", metric="RMSE")
+  control_model <- caret::train(x=x, y=y, trControl = train_control, method ="lm", metric="RMSE")
+
+  ffs_score <- as.double(ffs_model$results["RMSE"])
+  control_score <- as.double(control_model$results["RMSE"])
+
+  # Test
+
+  test_that('output should be a vector of double', {
+    expect_true(typeof(selected)=="double")
+  })
+
+  test_that('length of selected features should not be greater than the total number of features', {
+    expect_less_than(length(selected), length(x))
+  })
+
+  test_that('forward selected features should have the best score', {
+    expect_less_than(ffs_score, control_score)
+  })
```

Comment:
add tests to check if exception is being raised
we can implement this for next milestone

```suggestion
test_that('should throw error', {
    expect_error(ForwardSelection(my_mod="lm", feature=x, label=y, min_f='a', type="regression", cv=5), "min_f must be an integer")
  })
```